scripts/route_check.py

```diff
@@ -555,11 +634,22 @@ def check_routes():
     if rt_asic_miss:
         results["Unaccounted_ROUTE_ENTRY_TABLE_entries"] = rt_asic_miss
 
+    rt_frr_miss = check_frr_pending_routes()
```

Comment:
this should be invoked only if suppress fib feature is enabled. We dont need to execute it in normal flow.

Reply:
@prsunny This is done on purpose. Considering fpmsyncd has to reply back to zebra even when suppression is disabled (https://github.com/stepanblyschak/SONiC/blob/bgp-suppress-fib-pending/doc/BGP/BGP-supress-fib-pending.md#figure-5-fpmsyncd-flow) I though we need to add this check, regardless of the feature status.